examples/bno055_webgl_demo/static/css/bootstrap.css.map.license

```diff
@@ -0,0 +1,2 @@
+# SPDX-FileCopyrightText: 2021 ladyada for Adafruit Industries
```

Comment:
I am less sure about .map files I don't know if those are part of the core bootstrap project or if they were something added to this project. if they are core bootstrap I would think they should be copyright 2018 twitter also.